prime-router/docs/how-to-setup-an-apple-silicon-mac.md

```diff
@@ -0,0 +1,91 @@
+# Setup Apple Silicon Macs for Development
+
+## Problems
+
+At the time of writing this note, Microsoft's Azure Function docker container is not compatible with Apple Silicon processors. 
+Microsoft has not announced plans to fix this problem. 
+
+The current `./devenv-infrastructure.sh` runs the Frontend build in a container. 
+This container doesn't run because of the use of ARM64. 
+
+## Workarounds
+
+Fortunately, most of our tooling works on Apple Silicon, and it is possible to run ReportStream without a Docker container.  
+
+### Step 1 - Follow the getting started instructions
+
+Follow most of the [Getting Started](getting_started.md) instructions, including:
+
+1. Installing the recommend tools 
+2. Run `./cleanslate.sh`
+
+The step that runs the `./devenv-infrastructure.sh` will not work. 
+
+### Step 2 - Run dependencies
+
+ReportStream depends on set of containers to be up before running. Run these containers now using `docker-compose`.
+
+```bash
+docker-compose up sftp redox azurite vault web_receiver --detach
+```
+
+### Step 3 - Run ReportStream
+With the dependent services running, we can run ReportStream locally. 
+
+```bash
+gradle run
+```
+
+A `ctrl-c` will kill the running ReportStream process. 
+For your change, build, debug cycle, you can use the gradle to execute these steps. 
+
+```bash
+# to build the project
+gradle clean package fatjar
+
+# to run the build
+gradle run
```

Comment:
There is a step missing here.  You need to load the settings.  Run `gradle reloadSettings` while ReportStream is running.

Reply:
So far I haven't had to do this, maybe because of the call to `multiple-settings` below?